README.md

```diff
@@ -107,7 +102,10 @@ The package provides methods to easily listen for incoming messages and respondi
 - `bot.start_typing(receiver)`: Start typing
 - `bot.stop_typing(receiver)`: Stop typing
 - `bot.scheduler`: APScheduler > AsyncIOScheduler, see [here](https://apscheduler.readthedocs.io/en/3.x/modules/schedulers/asyncio.html?highlight=AsyncIOScheduler#apscheduler.schedulers.asyncio.AsyncIOScheduler)
-- `bot.storage`: In-memory or Redis stroage, see `storage.py`
+
+### Storage
+
+`bot.storage` is In-memory, SQLite,  or Redis storage, see `storage.py`. This is important because the bot runs multi-threaded, so instance state on your command classes isn't shared between threads. 
```

Comment:
```suggestion
`bot.storage` is in-memory, SQLite, or Redis storage, see `storage.py`.
This is important because the bot runs multi-threaded, so instance state on your command classes isn't shared between threads. 
```